service/create/service.go

```diff
@@ -243,7 +239,7 @@ func (s *Service) Boot() {
 						policy = &awsresources.Policy{
 							ClusterID: cluster.Spec.Cluster.Cluster.ID,
 							KMSKeyArn: kmsKey.Arn(),
-							S3Bucket:  s.s3Bucket,
+							S3Bucket:  cluster.Name,
```

Comment:
For my mental model. The S3 bucket is used to upload cloud configs to. For something else besides that? What about scoping the bucket also by organization/customer? That way we have all cloud configs of guest clusters associated to the organization/customer together.